### PR TITLE
Add support for Monzo

### PR DESCRIPTION
**Pull request by lwis**
Ref home-assistant/home-assistant#3747

### CODE REVIEW

File: source/_components/sensor.monzo.markdown [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      - local_currency
+      - local_exchange_rate
+```
+
+Configuration variables:
+
+- **access_token** (*Required*): Your access token for http://monzo.com/.
+- **account_id** (_Optional_): An account id, if omitted the first account will be used.
+- **monitored_variables** array (*Required*): Variables to display in the frontend.
+  - **balance**: The account balance.
```
Comment (Landrash) [inline source/_components/sensor.monzo.markdown]: Are all these required or optional.
Update them accordingly and try to minimize the configuration example.

Comment (Landrash) [inline source/_components/sensor.monzo.markdown]: Are all these required or optional.
Update them accordingly and try to minimize the configuration example.

Comment (lwis) [inline source/_components/sensor.monzo.markdown]: @Landrash I tried to follow https://home-assistant.io/components/sensor.darksky/ as an example, which lists all configurable parameters - I've opted not to include `account_id` in the example as it's optional.

Comment (Landrash) [inline source/_components/sensor.monzo.markdown]: @lwis Your correct. It looks fine then :)

Comment (lwis) [inline source/_components/sensor.monzo.markdown]: @Landrash cheers, bud.

**Pull request closed without merging**